core/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ActionManagerImpl.java

```diff
@@ -105,8 +106,10 @@ private void registerDefaultActionGroups() {
         DefaultActionGroup mainContextMenuGroup = new DefaultActionGroup(IdeActions.GROUP_MAIN_CONTEXT_MENU, false, this);
         registerAction(IdeActions.GROUP_MAIN_CONTEXT_MENU, mainContextMenuGroup);
 
-        DefaultActionGroup buildContextMenuGroup = new DefaultActionGroup(IdeActions.GROUP_BUILD_CONTEXT_MENU, false, this);
+        DefaultActionGroup buildContextMenuGroup = new DefaultActionGroup("Maven", true, this);
```

Comment:
We have the same behavior here: https://github.com/eclipse/che/blob/19162869b908fb61794518a3765a47a5380d7158/core/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ActionManagerImpl.java#L77-L101
I think we should have the consistent behavior.
So should we use constants in all places in this class?
@vparfonov  WDYT?